libraries/AP_Camera/AP_Camera.h

```diff
@@ -46,11 +47,28 @@ class AP_Camera {
     void            control(float session, float zoom_pos, float zoom_step, float focus_lock, float shooting_cmd, float cmd_id);
 
     // set camera trigger distance in a mission
-    void            set_trigger_distance(uint32_t distance_m)
+    void            set_trigger_distance(float distance_m)
     {
         _trigg_dist.set(distance_m);
     }
 
+    // CAMPOS methods
+    void            set_campos_parameters(uint8_t positions, float roll, float pitch, int16_t interval)
```

Comment:
```suggestion
    void set_campos_parameters(uint8_t positions, float roll, float pitch, int16_t interval)
```

Reply:
I will update that.

Reply:
I was following the spacing from the other methods. Actually I don't know why there are so many spaces in them.